SocialFilter/sections/specific.txt

```diff
@@ -4220,7 +4220,7 @@ ameblo.jp##._1JtRssPb
 4g63evo.net,shinoyon.com,win-tab.net,otakaranet.com,pokemongo-get.com,genda-yousuke.com,naoki440.info,npolaris.net,pasokatu.com,toolmania.info,gamegaz.com,nobitaka.com,mutyun.com,9ketsuki.info##.sns-buttons
 islandpacket.com##.share-tools-wrapper
 itmedia.co.jp##.wrap_social_big
-itmedia.co.jp###masterSocialbuttonTop
+itmedia.co.jp##.masterSocialbutton
```

Comment:
I think we should keep the print button

Reply:
@gork7777 
and please add `||nakanohito.jp^$third-party` to the tracking_servers.txt

Reply:
How do we keep it?

Reply:
@gork7777
Use this rule instead: `itmedia.co.jp##.msbBtnTweetbutton_minimal`

Reply:
@gork7777 Use `itmedia.co.jp##.msbBtnTweetbutton_minimal` instead

Reply:
It has social buttons both above and below the article. `###masterSocialbuttonTop` rule is blocking the above socials and also comment field too

Reply:
@gork7777 `itmedia.co.jp##.masterSocialbutton` blocks them too, please use `itmedia.co.jp##.msbBtnTweetbutton_minimal` instead of these rules:
```
itmedia.co.jp###masterSocialbuttonTop
itmedia.co.jp##.masterSocialbutton
```

Reply:
Shouldn't we separate the print and the comment? I was thinking there was a rule that we should separate them

Reply:
@gork7777
I don't really understand what you mean, we just shouldn't block the print and comment buttons.

Reply:
`###masterSocialbuttonTop` and my rule is blocking the above socials and comment. We need another rule to don't block comment and print. Also we need block below tweet.

Reply:
> We need another rule to don't block comment and print. Also we need block below tweet.

@gork7777
We achieve this result if we replace `itmedia.co.jp###masterSocialbuttonTop` with `itmedia.co.jp###.msbBtnTweetbutton_minimal`.

Please add these rules to user rules and check a result:
```
itmedia.co.jp##.msbBtnTweetbutton_minimal
itmedia.co.jp#@##masterSocialbuttonTop
```

Reply:
It's fine now

Reply:
@gork7777 we don't need this rule in the main filters
`itmedia.co.jp#@##masterSocialbuttonTop`

because we remove `itmedia.co.jp###masterSocialbuttonTop`